src/extensions/yfm/YfmTabs/utils.ts

```diff
@@ -17,3 +23,17 @@ export const atEndOfPanel = (view?: EditorView) => {
 
     return null;
 };
+
+export const switchTabByElem = (tabElem: HTMLElement) => {
+    if (tabElem.classList.contains('yfm-tab')) {
+        tabElem.click();
+    }
+};
+
+export const switchTabById = (container: HTMLElement, tabId: string) => {
+    const selector = `.yfm-tab[data-diplodoc-id=${tabId}]`;
+    const tabElem = container.querySelector<HTMLDivElement>(selector);
+    if (tabElem) {
+        switchTabByElem(tabElem);
+    }
+};
```

Comment:
I suggest moving the classes to constants

for example like this:
```
const CLASS_TAB = 'yfm-tab';
const ATTRIBUTE_ID = 'data-diplodoc-id';

export const switchTabByElem = (tabElem: HTMLElement) => {
    if (tabElem.classList.contains(CLASS_TAB)) {
        tabElem.click();
    }
};

export const switchTabById = (container: HTMLElement, tabId: string) => {
    const selector = `.${CLASS_TAB}[${ATTRIBUTE_ID}=${tabId}]`;
    const tabElem = container.querySelector<HTMLDivElement>(selector);
    if (tabElem) {
        switchTabByElem(tabElem);
    }
};
```

Reply:
fixed